CHANGELOG.md

```diff
@@ -77,15 +77,17 @@ All notable changes to this project will be documented in this file.
 * ### NI-Digital Pattern Driver
     * #### Added
         * `conditional_jump_triggers` and `pattern_opcode_events` repeated capabilities - [#1191](https://github.com/ni/nimi-python/issues/1191), [#1192](https://github.com/ni/nimi-python/issues/1192)
+        * `get_pattern_pin_names` - [#1292](https://github.com/ni/nimi-python/issues/1292)
```

Comment:
I think you need to fix the CHANGELOG integration so changes go in the unreleased section.